casbin/enforcer.cpp

```diff
@@ -444,7 +444,63 @@ bool Enforcer :: Enforce(Scope scope) {
     return this->enforce("", scope);
 }
 
+// Enforce with two params, decides whether a "subject" can do the operation "action", input parameters are usually: (sub, act).
+bool Enforcer::Enforce(string sub, string act) {
```

Comment:
If at all you are making the function for 
bool Enforce(string sub, string act);
then we also need, 
bool Enforce(string obj, string act);

Reply:
So we don't need this functions, if at all user need to use two param enforcement we can provide `bool Enforcer::Enforce(unordered_map<string, string> params);` method call for that purpose